server/backup_binlog_test.go

```diff
@@ -120,14 +120,25 @@ func testBackupBinlog() {
 		_, err := gsrv.FlushAndBackupBinlog(context.Background(), req)
 		Expect(err).ShouldNot(HaveOccurred())
 
+		By("checking in-progress metric is set")
+		binlogBackupGauge, _ := getMetric(registry, metricsPrefix+"binlog_backup_in_progress")
+		Expect(*binlogBackupGauge.Gauge.Value).Should(Equal(1.0))
+		Expect(*binlogBackupGauge.Label[0].Name).Should(Equal("cluster_name"))
+		Expect(*binlogBackupGauge.Label[0].Value).Should(Equal(clusterName))
+
+		By("wating binlog backup process is finished")
 		Eventually(func() error {
 			if agent.sem.TryAcquire(1) {
 				agent.sem.Release(1)
 				return nil
 			}
-			return errors.New("backup process is still working")
+			return errors.New("process is still working")
 		}, 30*time.Second).Should(Succeed())
 
+		By("checking in-progress metric is set")
```

Comment:
The word "cleared" is used in clone_test.go.

```suggestion
		By("checking in-progress metric is cleared")
```

Reply:
🙇🙇